src/io/flutter/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -262,17 +263,22 @@ public void scheduleConnect() {
       // because "flutter run" has already connected to it.
       final VmService vmService;
       final VmOpenSourceLocationListener vmOpenSourceLocationListener;
-      try {
-        vmService = VmService.connect(url);
-        vmOpenSourceLocationListener = VmOpenSourceLocationListener.connect(url);
-      }
-      catch (IOException | RuntimeException e) {
-        onConnectFailed("Failed to connect to the VM observatory service at: " + url + "\n"
-                        + e.toString() + "\n" +
-                        formatStackTraces(e));
-        return;
+      // TODO(github.com/flutter/flutter-intellij/issues/3293, github.com/dart-lang/webdev/issues/233) The following check disables the
```

Comment:
same as other cases. add a bug for the TODO

Reply:
actually in this case only list the one bug related to the issue and not the bug describing the feature you are implementing.

Reply:
done